front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -69,12 +71,18 @@ export default function Timetable({
   const [multiselectOn, setMultiselectOn] = useState(false);
   const [conflictsListExpanded, setConflictsListExpanded] = useState(false);
   const [selectedTrainIds, setSelectedTrainIds] = useState<number[]>([]);
+  const [rollingStockFilter, setRollingStockFilter] = useState('');
+  const [validityFilter, setValidityFilter] = useState('both');
+  const [isFilterPanelOpen, setIsFilterPanelOpen] = useState(false);
+  const [uniqueTags, setUniqueTags] = useState<string[]>([]);
+  const [selectedTags, setSelectedTags] = useState(new Set());
 
   const { openModal } = useContext(ModalContext);
 
   const dispatch = useDispatch();
 
-  const debouncedTerm = useDebounce(filter, 500) as string;
+  const debouncedFilter = useDebounce(filter, 500) as string;
+  const debouncedRollingstockFilter = useDebounce(rollingStockFilter, 500) as string;
```

Comment:
```suggestion
  const debouncedFilter = useDebounce(filter, 500);
  const debouncedRollingstockFilter = useDebounce(rollingStockFilter, 500);
```
I don't think you need this here :)

Reply:
Done